docs/rules/await-async-query.md

```diff
@@ -1,4 +1,4 @@
-# Enforce async queries to have proper `await` (await-async-query)
+# Enforce promise from async queries to be handled (await-async-query)
```

Comment:
extremely minor comment, I think it should be plural

```suggestion
# Enforce promises from async queries to be handled (await-async-query)
```

because `queries` is plural